src/main/java/org/example/racinggame/controller/RacingCarController.java

```diff
@@ -0,0 +1,38 @@
+package org.example.racinggame.controller;
+
+import org.example.racinggame.model.Cars;
+import org.example.racinggame.view.InputView;
+import org.example.racinggame.view.OutputView;
+
+public class RacingCarController {
+
+    private final InputView inputView;
+    private final OutputView outputView;
+
+    public RacingCarController(InputView inputView, OutputView outputView) {
+        this.inputView = inputView;
+        this.outputView = outputView;
+    }
+
+    public void play() {
+        String carNames = inputView.getCarNamesFromUserInput();
+        int tryCount = inputView.getTryCountFromUserInput();
+
+        Cars cars = new Cars(carNames);
+
+        outputView.printResult();
+        for (int i = 0; i < tryCount; i++) {
+            cars.move();
+            printCarPositions(cars);
+        }
+
+        printCarPositions(cars);
+        outputView.printWinners(cars.selectWinners());
+    }
+
+    private void printCarPositions(Cars cars) {
+        cars.getCarList()
+                .forEach(car -> outputView.printCarPosition(car.getName(), car.getPosition()));
+        outputView.newLine();
```

Comment:
아래와 같이 `List<Car>`를 outputView의 메서드로 넘겨서 outputView에서 콘솔 출력 로직을 수행해도 좋을 것 같아요. 😃

```suggestion
        outputView.printCarPositions(cars.getCarList());
```